cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -149,6 +117,10 @@ func (c bundleCmd) validateManifests(*config.Config) (err error) {
 		}
 	}
 
+	if c.kustomizeDir == "" {
+		return errors.New("--kustomize-dir must be set")
```

Comment:
So `--kustomize-dir` seems to be a required flag now. Unless it should be defaulted to `config/manifests`.
Can we mark it as such, and also I didn't see it used in the example text:
```
$ kustomize build config/manifests | operator-sdk generate bundle --manifests --metadata --overwrite --version 0.0.1
```

Reply:
This flag is required but [defaults to](https://github.com/operator-framework/operator-sdk/pull/3257/files#diff-0aff0dcd8bfc140fd01b40d46a1f7bc1R116) `config/manifests`. I will mark it as required.

Reply:
Just double checking. It is required in the sense that it can't be empty, but not required in the sense that it doesn't have to be provided on the command line?

Reply:
Correct, much like `--deploy-dir` in the legacy CLI.